src/libstrongswan/utils/identification.c

```diff
@@ -67,8 +67,7 @@ static const x501rdn_t x501rdns[] = {
 	{"UID", 				OID_PILOT_USERID,			ASN1_PRINTABLESTRING},
 	{"DC", 					OID_PILOT_DOMAIN_COMPONENT, ASN1_PRINTABLESTRING},
 	{"CN",					OID_COMMON_NAME,			ASN1_PRINTABLESTRING},
-	{"S", 					OID_SURNAME,				ASN1_PRINTABLESTRING},
-	{"SN", 					OID_SERIAL_NUMBER,			ASN1_PRINTABLESTRING},
+	{"SN", 					OID_SURNAME,				ASN1_PRINTABLESTRING},
```

Comment:
https://github.com/strongswan/strongswan/commit/a8c09d8cc0044fbff6c89c2e8966427ee7ed1ef0#diff-019ea69dc5521c8a94784efcbc8478c7R72

The bug was introduced 14 years ago !